source/about/cloud-subscriptions.rst

```diff
@@ -59,14 +115,24 @@ After your first month, the billing period begins at 12 AM UTC on the first day
 
 - If you begin your subscription in the middle of a billing period, charges will be prorated. For example, if you signed up 6/15, we'll charge you on 7/1 for the period 6/15 - 6/30.
 - Similarly, if you end your subscription in the middle of a billing period, charges will be prorated. For example, if you cancel your account on 6/15, we'll charge you on 7/1 for the period 6/1 - 6/15.
-- If you upgrade in the middle of a billing period, charges will also be prorated. For example, if you upgrade from Starter to Professional on 6/15, we'll charge you 7/1. This will be a single charge for Starter for the period 6/1 - 6/14, then a separate charge for Professional for the period 6/15 - 6/30.
+- If you upgrade in the middle of a billing period, charges will also be prorated. For example, if you upgrade from Professional to Enterprise on 6/15, we'll charge you 7/1. This will be a single charge for Professional for the period 6/1 - 6/14, then a separate charge for Enterprise for the period 6/15 - 6/30.
+
+What happens if I decide to purchase a subscription to Mattermost Enterprise during my trial?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your plan immediately changes to the plan you've upgraded to. Your trial will continue and on day 31 billing will commence. You won't incur any charges of your new plan until the trial period is over. Depending on what day of the month this is, you'll be billed pro-rata. You'll receive your bill at the end of the calendar month for this month and going forward.
+
+How do I change my subscription from Mattermost Enterprise to Mattermost Professional?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can downgrade on https://mattermost.com/pricing and select Mattermost Professional.
```

Comment:
Actually to do this they'll have to go change their plan in-product in the System Console > Subscription and clicking "View plans", where they can change their plan. **However**, this has not been implemented yet.

This is where "View plans" will lead them to:
<img width="764" alt="Screen Shot 2022-06-13 at 8 18 12 PM" src="https://user-images.githubusercontent.com/101303489/173467883-b7220b5d-0709-40a4-a428-737e4db3671c.png">


For now, can we just tell them to go contact something like support@mattermost.com? Once we have the "View plans" button available for Enterprise/Professional (available soon), we can update this.

What do you think?

Reply:
That sounds reasonable to me!